Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -52,18 +42,59 @@ def family
             :cannonlake
           when 0x6a, 0x6c, 0x7d, 0x7e
             :icelake
-          else
-            unknown
           end
         when 0x0f
-          case cpu_model
+          case model
           when 0x06
             :presler
           when 0x03, 0x04
             :prescott
-          else
-            unknown
           end
+        end
+      end
+
+      def amd_family(family)
+        case family
+        when 0x06
+          :amd_k7
+        when 0x0f
+          :amd_k8
+        when 0x10
+          :amd_k10
+        when 0x11
+          :amd_k8_10_hybrid
+        when 0x12
+          :amd_k12
+        when 0x14
+          :bobcat
+        when 0x15
+          :bulldozer
+        when 0x16
+          :jaguar
+        when 0x17
+          :zen
+        when 0x19
+          :zen3
+        end
+      end
+
+      def family
```

Comment:
Can this method stay where it was above so we can review changes more easily?

Reply:
moved the method back up!